### DISCUSSION TITLE
Get SellerSKU for fees like FBAStorageFee and FBAInboundTransportationFee

### DISCUSSION THREAD

**Discussion by niccoloalexander** [amzn/selling-partner-api-models#3704] [General]
When I get the above fee events, the SellerSKU always returns 'null'. Is there any way to get the attributed SKU? When checked on Seller Central, I'm able to find these costs attributed to specific SKUs.

**Comment by rugved1991** [comment]
Hi @niccoloalexander,

Can you please provide more context on what API or Notification you are referring to? Can you tell me more about the use case you are trying to achieve?

Best,
Rugved
Solutions Architect, SP-API

**Comment by niccoloalexander** [comment]
Hi @rugved1991, we are collecting via the [Finances API](https://developer-docs.amazon.com/sp-api/docs/finances-api-reference#adjustmenteventlist). The FBAStorageFee and FBAInboundTransportationFee events, among others, appear to always have a null SellerSKU. Is there a way to get these events with the SKU attributed? 

The following lists appear to not provide SellerSKUs with their corresponding fee events
- CouponPaymentEventList
- ServiceFeeEventList

The use case is to visualize a breakdown of finances at the SKU-level for all SKUs at a time. Where the total of SKU-level profit and loss equals my account profit and loss total.

**Comment by anwarmuhammad** [comment]
@niccoloalexander have you solve this issue? I am also facing same issue .
I will appreciate if anybody in the community can answer it .
Thanks

**Reply by niccoloalexander** [reply]
No answer yet @anwarmuhammad . Would be helpful to have a direct SKU attribution.

**Reply by syedhsan** [reply]
@niccoloalexander  hey bro have find any solution for this I am facing same issue to get SKU of products

**Comment by IsaacSee** [comment]
Hi, @rugved1991 can you please get back to us with a solution or some direction? Any help here will be appreciated, thanks!

**Comment by ArthurDemu** [comment]
Having the same issue here, would really help to have a SKU attributed to events like storage and removal fees

**Comment by syedhsan** [comment]
@niccoloalexander 
@IsaacSee @anwarmuhammad @rugved1991 
Hey all have you find the solution related to this I m facing same issue of attributes SKU
